libs/thunk/windows/platform.cpp

```diff
@@ -16,10 +15,9 @@ class _invalid_parameter_suppression
   public:
     _invalid_parameter_suppression()
     {
-        _CrtSetReportMode(_CRT_ASSERT, 0);
```

Comment:
Won't removing this will cause the CRT to popup a modal dialog box during the tests, causing the CI/CD to hang?